app/build.gradle.kts

```diff
@@ -190,6 +201,18 @@ dependencies {
         // Networking with OkHttp
         implementation(libs.okhttp)
 
+    androidTestImplementation("androidx.test:runner:1.4.0")
+    androidTestUtil("androidx.test:orchestrator:1.4.1") // Use Android Test Orchestrator
+
+
+
+
+}
+
+configurations.all {
+    resolutionStrategy {
+        force("androidx.test.espresso:espresso-core:3.6.1")
+    }
```

Comment:
Conflict with main branch: Unresolved merge between androidTest dependencies and calendar dependencies. Please ensure both `androidTestImplementation` and calendar dependencies are properly merged to maintain the required functionality.